docs/about/contributing.md

```diff
@@ -49,9 +49,16 @@ to the git repository.
 
 ## Running the tests
 
-To run the tests, it is recommended that you use [Tox]. This just needs
-to be pip installed and then the test suite can be ran for MkDocs but running
-the command `tox` in the root of your MkDocs repository.
+To run the tests, it is recommended that you use
+[Tox](https://pypi.org/project/tox/).
+
+Install Tox using [pip](https://pypi.org/project/pip/) by running the
```

Comment:
Please follow the pattern for links used elsewhere in the document. All links are reference style links with the URLs grouped together at the bottom of the document. In fact, there is already a link for `tox` there. No need to repeat the URL again. Also, why are we using pypi links? The projects themselves don't advertise those links. Lets use the links which the projects themselves use.

Reply:
Fixed in ccdd882 (updated link to `pip`).